application/src/test/java/org/opentripplanner/apis/gtfs/service/ApiTransitServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package org.opentripplanner.apis.gtfs.service;
+
+import static com.google.transit.realtime.GtfsRealtime.TripDescriptor.ScheduleRelationship.SCHEDULED;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.transit.model._data.TimetableRepositoryForTest.id;
+import static org.opentripplanner.updater.spi.UpdateResultAssertions.assertSuccess;
+import static org.opentripplanner.updater.trip.UpdateIncrementality.FULL_DATASET;
+
+import com.google.transit.realtime.GtfsRealtime;
```

Comment:
Using this dependency here feels slightly wrong but I don't know if there is another way to do it?

Reply:
I could manipulate the timetable repository manually. That's very complicated. You tell me if it's worth it.